torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -3036,8 +3036,21 @@ at::Tensor XLANativeFunctions::softshrink_backward(const at::Tensor& grad_out,
 std::tuple<at::Tensor, at::Tensor> XLANativeFunctions::sort(
     const at::Tensor& self, int64_t dim, bool descending) {
   XLA_FN_COUNTER("xla::");
-  auto results = XLATensor::topk(bridge::GetXlaTensor(self), self.size(dim),
-                                 dim, descending, true);
+  auto results =
+      XLATensor::topk(bridge::GetXlaTensor(self), self.size(dim), dim,
+                      descending, /*sorted=*/true, /*stable=*/false);
```

Comment:
Do we know why we pass `/*sorted=*/true` here? I am not sure topk actually do anything for pt/xla.

Reply:
Hey @JackCaoG , the original `XLATensor::topk` does provide `sorted` as in https://pytorch.org/docs/stable/generated/torch.topk.html. However, in ptxla, we don't really do anything with `sorted ` and always returns sorted results in `CreateTopK`. Should we create a separate PR to restore/introduce a meaningful `sorted` flag for `topk` ?